app/app.rb

```diff
@@ -0,0 +1,91 @@
+ENV['RACK_ENV'] ||= "development"
+
+require 'sinatra/base'
+require 'bcrypt'
+require 'data_mapper'
+require_relative './models/user'
+require_relative './models/peep'
+require 'sinatra/flash'
+
+class Chitter < Sinatra::Base
+
+	register Sinatra::Flash
+	enable :sessions
+	set :session_secret, 'super secret'
+
+	use Rack::MethodOverride
+  
+  get '/' do
+  	redirect '/sign_up'
+  end
+
+  get '/sign_up' do
+  	@user = User.new
+    erb(:'users/sign_up')
+  end
+
+  post '/users' do
+  	@user = User.create(email: params[:email],
+                  name: params[:name],
+                  nickname: params[:nickname],
+  								password: params[:password],
+  								password_confirmation: params[:password_confirmation])
+  	if @user.save # #save returns true/false depending on whether the model is successfully saved to the database.
+	    session[:user_id] = @user.user_id
+	    redirect to('/peep')
+
+  	else
+			flash.now[:errors] = @user.errors.full_messages   		
+			erb :'users/sign_up'
+		end
+  end
+
+  get '/peep' do
+    @peeps = Peep.all
+   	erb :'peep/peep_list'
+  end
+
+  get '/peep/new' do
+    if session[:user_id]
+      erb :'peep/peep_new'
+    else
+      flash.now[:errors] = ['Sign in for peeping']
+      erb :'users/sign_in'
+    end
+  end
+
+  post '/peep' do
+    Peep.create(text: params[:peep])
+    redirect to('/peep')
+  end
+
+
+  get '/sign_in' do
+  	erb :'users/sign_in'
+	end
```

Comment:
Lint/BlockAlignment: `end` at 65, 1 is not aligned with `get '/sign_in' do` at 63, 2.